index.html

```diff
@@ -30,11 +31,44 @@
             <h2>Bezalell</h2>
           </a>
         </div>
-        <div class="bars">
-          <div class="bar1"></div>
-          <div class="bar2"></div>
-          <div class="bar3"></div>
+        <div class="menu-btn">
+          <div class="btn-line"></div>
+          <div class="btn-line"></div>
+          <div class="btn-line"></div>
         </div>
+        <nav class="menu">
+          <div class="menu-branding">            
+          </div>
+          <ul class="menu-nav">
+            <li class="nav-item current">
+              <li class="nav-item">
+                <a href="index.html" class="nav-link">
+                  Home
+                </a>
+                <hr class="underline">
+              </li>
+              <li class="nav-item">
+                <a href="about.html" class="nav-link">
+                  Portfolio
+                </a>
+                <hr class="underline">
+              </li>
+              <li class="nav-item">
+                <a href="work.html" class="nav-link">
+                  About
+                </a>
+                <hr class="underline">
+              </li>
+              <li class="nav-item">
+                <a href="contact.html" class="nav-link">
+                  Contact
+                </a>
+                <hr class="underline">
+              </li>
```

Comment:
- [ ] kindly consider implementing the javascript  functionalities of links in your nav  so that if a user clicks on any of them it just take him/her on the right part of your portfolio it will also help you to fulfill the below requirements
>- When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed.

> - When the user clicks (or taps) any option of the mobile menu, the mobile menu disappear.